--- /lib/systemd/system/nginx.service.orig      2018-11-16 09:24:45.643260358 +0000
+++ /lib/systemd/system/nginx.service   2018-11-16 09:25:09.587621593 +0000
@@ -17,6 +17,7 @@
 
 [Service]
 Type=forking
+EnvironmentFile=/opt/portier/nginx/nginx-env
 PIDFile=/run/nginx.pid
 ExecStartPre=/usr/sbin/nginx -t -q -g 'daemon on; master_process on;'
 ExecStart=/usr/sbin/nginx -g 'daemon on; master_process on;'
